main/tasks.py

```diff
@@ -146,3 +147,44 @@ def notify_monthly_time_logged_summary() -> None:
         current_month_start,
         current_month_name,
     )
+
+
+_template_charges_report = """
+Dear {HoRSE},
+
+Please find attached the charges report for the last month: {month}.
+
+Best regards,
+ProCAT
+"""
+
+
+def email_monthly_charges_report_logic(month: int, year: int, month_name: str) -> None:
+    """Logic to email the HoRSE the charges report for the last month."""
+    from .models import User
+
+    HoRSE = User.objects.filter(is_superuser=True)[0]  # Assuming there is 1 superuser
```

Comment:
I guess there might be more than one superuser, probably 1 more. Maybe we should recover this issue https://github.com/ImperialCollegeLondon/proCAT/issues/179

Let's leave it like that, for now.